### PR TITLE
[sensor_ctrl] Move to D2

### PR DESCRIPTION
**Pull request by tjaychen**
- also add missing checklist

Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

**Comment by tjaychen** [comment]
sorry forgot to push the PR yesterday.

**Review by eunchan — APPROVED** [review]

**Review by msfschaffner — APPROVED** [review]

File: hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
+---
+title: "SENSOR_CTRL Checklist"
+---
+
+<!--
+NOTE: This is a template checklist document that is required to be copied over to the 'doc'
+directory for a new design that transitions from L0 (Specification) to L1 (Development)
+stage, and updated as needed. Once done, please remove this comment before checking it in.
+-->
```
Comment (msfschaffner) [inline hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md:9]: you may want to remove this comment here

Comment (tjaychen) [reply hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md:9]: ah yes.

File: hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+[NO_HIGH_PRIORITY_ISSUES_PENDING]:    {{<relref "/doc/project/checklist.md#no_high_priority_issues_pending" >}}
+[ALL_LOW_PRIORITY_ISSUES_ROOT_CAUSED]:{{<relref "/doc/project/checklist.md#all_low_priority_issues_root_caused" >}}
+[DV_DOC_TESTPLAN_REVIEWED]:           {{<relref "/doc/project/checklist.md#dv_doc_testplan_reviewed" >}}
+[V3_CHECKLIST_SCOPED]:                {{<relref "/doc/project/checklist.md#v3_checklist_scoped" >}}
+
+### V2S
+
+ Type         | Item                                    | Resolution  | Note/Collaterals
+--------------|-----------------------------------------|-------------|------------------
+Documentation | [SEC_CM_TESTPLAN_COMPLETED][]           | Not Started |
```
[the remark this answers is not in the document]
Comment (sriyerg) [reply hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md:230]: Or better yet - delete the V checklist tables entirely and replace with a note.

[the remark this answers is not in the document]
Comment (tjaychen) [reply hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md:230]: sounds good, let me do that.

[the remark this answers is not in the document]
Comment (tjaychen) [reply hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md:230]: yes let me change to N/A also in the high level description instead of 'v0'

**Pull request merged**